src/features/dashboard/AccountSlice.ts

```diff
@@ -0,0 +1,241 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { onboard, testnet_addresses } from '../../utils/web3';
+import { ethers, Contract, formatUnits } from 'ethers'
+
+// ABIs
+import ERC20Immutable from '../../abis/Erc20Immutable.json'
+import Comptroller from '../../abis/Comptroller.json'
+interface AccountState {
+    loading: boolean;
+    error: string;
+    status: string,
+    liquidity: number,
+    borrowLimit: number,
+    borrowLimitUsed: number,
+    netAPR: number,
+    netBorrowBalance: number,
+    netSupplyBalance: number,
+}
+
+const initialState: AccountState = {
+    loading: false,
+    error: "",
+    status: "initial",
+    liquidity: 0e18,
+    borrowLimit: 0,
+    borrowLimitUsed: 0,
+    netAPR: 0,
+    netBorrowBalance: 0,
+    netSupplyBalance: 0
+}
+
+// Views
+
+export const updateNetSupplyBalance = createAsyncThunk('netSupplyBalance/update', async () => {
+    const [wallet] = onboard.state.get().wallets;  
+})
+
+export const updateNetBorrowBalance = createAsyncThunk('netBorrowBalance/update', async () => {
+    const [wallet] = onboard.state.get().wallets;
+})
```

Comment:
[Question]

Are these asynchronous? They both call the method to get the wallet so why do we need two separate thunks?

Reply:
These functions are not yet implemented!

Reply:
Then is the PR actually finished? At least put a TODO and tag it for the next ticket that way I would know what to review

Reply:
Yea the PR is finished,  we're just getting the visuals down